datalad/downloaders/tests/test_s3.py

```diff
@@ -120,3 +130,31 @@ def deny_access(*args, **kwargs):
     with assert_raises(DownloadError):
         with patch.object(downloader, '_download', deny_access):
             downloader.download("doesn't matter")
+
+
+@with_tempfile
+def test_boto_host_specification(tempfile):
+    # This test relies on a yoh-specific set of credentials to access
```

Comment:
While this of course is useful on your end, my preference would be to not bring this into the main tree.

Reply:
where/how to keep it so I don't forget and test whenever I do change anything in related code?

Reply:
If having an appropriately named local branch isn't enough, you could leave a one-line comment in this file as a note for yourself.  Anyway, just my two cents.  Feel free to disregard.